helmfile/values/dex.yaml.gotmpl

```diff
@@ -34,6 +39,10 @@ extraVolumes:
 {{- end }}
 
 config:
+  storage:
+    type: kubernetes
+    config:
+      inCluster: true
```

Comment:
I think this should be `type: memory` as per the ADR linked in the issue: https://github.com/elastisys/compliantkubernetes-apps/issues/385#issuecomment-854756865

Reply:
I don't remember us talking about this, so maybe it's already answered by @cristiklein. But [the documentation](https://dexidp.io/docs/storage/) states

> Dex requires persisting state to perform various tasks such as track refresh tokens, preventing replays, and rotating keys

I feel like at least "preventing replays" would be something we would like to defend ourselves from?

Reply:
I think (but I'm not sure) that dex persists this in memory with `type: memory` and if dex then is restarted you have to log in again (so we are safe from replays, etc).

Reply:
Right, so it'll mainly be a bit annoying rather than a security risk. That seems fine.

And also I can't find the memory storage type in the docs. is it `type: sqlite3` you mean?

Reply:
No I think it is `type: memory`, see this https://github.com/dexidp/helm-charts/tree/master/charts/dex#minimal-configuration and this https://github.com/dexidp/dex/blob/master/storage/memory/memory.go
But I have also seen that it is missing from the docs, and I have not tested it myself.